data/json/furniture.json

```diff
@@ -598,7 +599,8 @@
             "items": [
                 { "item": "scrap", "count": [2, 6] },
                 { "item": "steel_chunk", "prob": 50 },
-                { "item": "pipe", "count": 1 }
+                { "item": "pipe", "count": 1 },
+                { "item": "lead", "charges": [500, 1000] }
```

Comment:
50% change of losing of steel chunk is reasonable (because you actually able to tear it by brute deconstruction) but I'm not sure it's applicable to lead, because we don't lose matter due to deconstruction (deconstruction is not annihilation after all)